dnf-behave-tests/features/repoquery/modules.feature

```diff
@@ -0,0 +1,22 @@
+Feature: repoquery tests for handling modularity.
+
+Background:
+ Given I use repository "repoquery-modules"
+
+
+Scenario: Test --disable-modular-excludes
```

Comment:
Can you please fix the option name here? Should be ``--disable-modular-filtering``.